radar.lua

```diff
@@ -104,10 +104,10 @@ s_protect.command_radar = function(name)
 			minetest.formspec_escape("[combine:300x300"
 				.. parts .. marker_str)
 				.. dir_mod .. "]" ..
-		"label[0,6.8;1 square = 1 area = "
+		"label[0,6.8;1 " .. S("square") .. " = 1 " .. S("area") .. " = "
```

Comment:
You could simply translate the entire thing.
`S("1 square = 1 area = @1x@2x@3 nodes (X,Y,Z)")` (or similar)

Reply:
Good idea, modif ok in my repository